### DISCUSSION TITLE
What kind of framework starters for Tailwind Elements do you need?

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1443] [Vote in polls]
We'll be creating integration guides soon & possibly separate fully integrated packages in the future.
Let us know **what should have the highest priority**.

**Update 2:**

The official dedicated package for React is already published:
https://tailwind-elements.com/docs/react/

But we're keeping the vote up in order to see which other guides should be transformed into dedicated packages first.

**Update:**

The official integration guides are already published!

- [React](https://tailwind-elements.com/docs/standard/integrations/react-integration/)
- [Remix](https://tailwind-elements.com/docs/standard/integrations/remix-integration/)
- [Vue](https://tailwind-elements.com/docs/standard/integrations/vue-integration/)
- [Angular](https://tailwind-elements.com/docs/standard/integrations/angular-integration/)
- [Svelte](https://tailwind-elements.com/docs/standard/integrations/svelte-integration/)
- [Next](https://tailwind-elements.com/docs/standard/integrations/next-integration/)
- [Nuxt](https://tailwind-elements.com/docs/standard/integrations/nuxt-integration/)

But we're keeping the vote up in order to see what kind of dedicated packages should we prepare first.

**Comment by bhuphathi** [comment]
solidjs

**Comment by kulame** [comment]
solidjs too

**Comment by witmarr97** [comment]
React + Next.js

**Comment by hdayi** [comment]
No framework or svelte

**Comment by LijahD99** [comment]
TALL stack

**Comment by v3ss0n** [comment]
Need more love for svelte.

**Comment by HobbyHaus** [comment]
Typescript

**Comment by joeyAddy** [comment]
Definitely React => (Next js)

**Comment by BillyAlbornoz** [comment]
Sveltekit!!!!!

**Reply by v3ss0n** [reply]
Yes !

**Comment by ghost** [comment]
Vue!!!!

**Comment by makikata** [comment]
Ruby on Rails

**Comment by lucWhiltech** [comment]
html and react

**Comment by agriffard** [comment]
Astro

**Comment by jemm88** [comment]
Flask

**Comment by keyse** [comment]
Blazor

**Comment by DanielGomez294** [comment]
react

**Reply by filipkappa** [reply]
Hello Daniel - did you know that:

1. TE already has a dedicated React version: https://tailwind-elements.com/docs/react/
2. TE has a detailed React integration guide: https://tailwind-elements.com/docs/standard/integrations/react-integration/#section-react-app

**Comment by Amstrong1** [comment]
Laravel

**Reply by jayenn007** [reply]
Can't get the laravel integration to work other than with CDN. Maybe you can add a Laravel/Sail integration guide?

**Comment by antik1801** [comment]
a

**Comment by MartyNZ** [comment]
Nuxt

**Comment by looch** [comment]
**HTML + Vanilla JavaScript/TypeScript !!!!!**

**Reply by BenWhite27** [reply]
Same for me, I just want to add some interactivity to a standard ASP.NET Core site.  Achieving this without an opinionated design like bootstrap is great.  I'd prefer not to deal with JavaScript modules etc, just add a JS script to the page and apply the required markup.

**Comment by daedcpl** [comment]
Nuxt3 :)

**Comment by nehathakur2021** [comment]
laravel

**Comment by nehathakur2021** [comment]
laravel+livewire

**Comment by davem1012** [comment]
livewire

**Comment by Jandro877** [comment]
TALL (laravel+livewire)

**Comment by WCN-llc** [comment]
Material Design 3

**Comment by AlanMcCann** [comment]
AlpineJS
(for Petal Stack)
Plain js is fine too

**Comment by dntzhang** [comment]
I am working on the omi version of tw elements, coming soon. http://omijs.org/

**Reply by filipkappa** [reply]
Awesome :D let us know when it's out!

**Reply by dntzhang** [reply]
Many components have been completed, I will let you know as soon as I'm done. I will put the entrance on the official website then.

**Reply by dntzhang** [reply]
@filipkappa The components are completed. Other parts are also ongoing.
github: https://github.com/Tencent/omi/tree/master/packages/omi-elements
preview: https://omi.cdn-go.cn/elements/latest/#/

**Reply by dntzhang** [reply]
Aligning the site layout with TW Elements site.
<img width="1726" alt="image" src="https://github.com/mdbootstrap/TW-Elements/assets/7917954/289c9928-3d7e-4447-9c42-ba4dd94b8e2b">
preview: https://omi.cdn-go.cn/elements/latest/#/

**Comment by Csb-218** [comment]
next.js

**Comment by rachiddri** [comment]
Laravel 10

**Comment by WalkerWang-UNIS** [comment]
Angular

**Comment by nheffelman** [comment]
I'm developing on django-tailwind.... So any help entegrating elements with that would be great!

**Reply by juujisai** [reply]
Hi! We have prepared an integration tutorial for TW Elements with Django. Checkout our docs here: https://tw-elements.com/docs/standard/integrations/django-integration/

**Comment by nheffelman** [comment]
Thanks a lot for the quick response.

**Comment by Kokoserver** [comment]
Python esmerald

**Comment by geemang2022** [comment]
DJANGO, HTMX and possibly Alpine.js

**Comment by hinagiku** [comment]
rails

**Comment by zikou058** [comment]
react + tailwindcss
